Abracadabra/abracadabra_web_app/src/components/SubjectQuestionAnswer.js

```diff
@@ -25,46 +30,43 @@ export default function Answer({ answer }) {
     AnswerId: answer.id,
     vote: ""
   })
-  
-    //#region FlaggingQuestion
+
+  //#region FlaggingQuestion
   const [showToast, setShowToast] = useState(false);
   const [toastText, setToastText] = useState("");
   const [toastColor, setToastColor] = useState("");
 
   const HandleAnswerFlagClick = async () => {
     ReportService.FlagAnswer(answer.id)
-    .then(res => {
-      //console.log(res);
-      setToastText("Answer successfully reported");
-      setToastColor("bg-success");
-      setShowToast(true);
-    })
-    .catch(err => {
-      //console.log(err);
-      try {
-      if(err.response.status == 400)
-      {
-        setToastText(err.response.data);
-        setToastColor("bg-warning");
-      }
-      else if(err.response.status == 401)
-      {
-        setToastText("You need to be logged in to report an answer!");
-        setToastColor("bg-warning");
-      }
-      else 
-      {
-        setToastText("Oops! Something went wrong with the API, try again later.");
-        setToastColor("bg-danger");
-      }
-      setShowToast(true);
-    }
-    catch {
-      setToastText("Oops! couldn't reach the report API, try again later.");
-      setToastColor("bg-danger");
-      setShowToast(true);
-    }
-    });
+      .then(res => {
+        //console.log(res);
+        setToastText("Answer successfully reported");
+        setToastColor("bg-success");
+        setShowToast(true);
+      })
+      .catch(err => {
+        //console.log(err);
+        try {
+          if (err.response.status == 400) {
+            setToastText(err.response.data);
+            setToastColor("bg-warning");
+          }
+          else if (err.response.status == 401) {
+            setToastText("You need to be logged in to report an answer!");
+            setToastColor("bg-warning");
+          }
+          else {
+            setToastText("Oops! Something went wrong with the API, try again later.");
```

Comment:
I think it would be better not to use technical terms such as API in error messages displayed for the user.

Same for all the others below